packages/issuer-api-client/src/api.ts

```diff
@@ -0,0 +1,1463 @@
+// tslint:disable
```

Comment:
Does the openapi generator tool provides any guidelines for storing those generated files? Typical approach is to keep autogenerated sources outside of git repository.

Reply:
I'll look into better ways, but I reckon we would then probably need to keep `@energyweb/issuer-api` as the dependency to `@energyweb/origin-ui-core`.

Reply:
Having the dependency of `@energyweb/issuer-api` to `@energyweb/origin-ui-core` would be bad, I think the main reason of having the issuer api client is the decouple these two.

Reply:
Yes, we definitely need to avoid that.